src/common/Icon.tsx

```diff
@@ -18,6 +18,7 @@ export type IconTypes =
   | "widgetAddOutline"
   | "downloadBold"
   | "closeCircleBold"
+  | "arrowRight"
```

Comment:
이름 뒤에 Bold를 추가하면 좋을 것 같아요.
나중에 같은 아이콘에 Outline 버전이 들어올 수도 있어서요~!
```suggestion
  | "arrowRightBold"
```